commit/merkleroot/types.go

```diff
@@ -73,10 +80,14 @@ func aggregateObservations(aos []plugincommon.AttributedObservation[Observation]
 				append(aggObs.OffRampNextSeqNums[seqNumChain.ChainSel], seqNumChain.SeqNum)
 		}
 
+		// RMNRemoteConfig
+		aggObs.RMNRemoteConfigs = append(aggObs.RMNRemoteConfigs, obs.RMNRemoteConfig)
```

Comment:
The nodes that can't read the destination chain will return empty `rmntypes.RMNRemoteConfig{}` objects, should we filter those out before aggregating?

Reply:
We saw this issue with discovery where nil slices were not filtered out.

Reply:
I wonder if this issue applies to all the aggregateObservation functions...

Reply:
I am not sure about the other element of the observation but I added an if condition to check whether the RMNRemoteConfig is empty for now. It makes sense to filter out the nil values.